TheForceEngine/TFE_FileSystem/fileutil-posix.cpp

```diff
@@ -110,6 +113,23 @@ namespace FileUtil
 
 	void getExecutionDirectory(char *dir)
 	{
+#ifdef __APPLE__
```

Comment:
If you replace the whole function, I'd prefer if you just created a new one and wrap the 2 versions in the #ifdefds:  I find that easier to review.

#ifdef __APPLE__
void getExecutionDirectory()
{ --- apple bits ---}

#else /* posix */

void getExecutionDirectory()
{ --- existing bits ---}

#endif